tests/utils.go

```diff
@@ -2883,49 +2883,29 @@ func configureIPv6OnVMI(vmi *v1.VirtualMachineInstance, expecter expect.Expecter
 		return err == nil
 	}
 
-	dnsServerIP, err := getClusterDnsServiceIP(virtClient)
-	if err != nil {
-		log.DefaultLogger().Object(vmi).Infof("Couldn't get DNS Service IP while configuring ipv6: %v", err)
-		expecter.Close()
-		return err
-	}
```

Comment:
as we said, we are removing the DNS from here, whoever needs the DNS will do it with cloud-init, 
the code that i am removing actually doesnt work (it doesnt set nameserver), and not needed, i also fix the problems on the expect there in the 2nd commit that i will squash at the end as we said, it still wip, it works, but i am checking one more thing.

the condition you speak on is in the same function. cirros doesnt have userdata so its the one that adding the dns, fedora ones that need dns added it via user-data
we dont have other that do need it (otherwise it would fail , because the backticks, not the sudo su)

Reply:
I think I've expressed my self poorly: I am asking about **routes** and the **IP address** configuration, not DNS. I'm well aware we've agreed to remove DNS configuration via console, leaving cloud-init as the only option to do configure DNS in the guests.

We read the IP of the DNS service for 2 reasons: 
  1. [config DNS in the guest](https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L2913)
  2. [know if](https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L2893) we're on an IPv6 cluster

Here, I'm concerned about the second point; if you remove this condition, we will end up configuring an ipv6 address + route on guests created on ipv4 clusters, which, while *probably* safe, is **not** required. 

Keeping the code that retrieves the cluster DNS service IP, and also keeping it [in the condition](https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L2893) would be preferable, IMHO, since guests on IPv4 clusters do not need - at all - that IPv6 address. I find it preferable to stick to the minimal configuration that works, since the tests can be used as references in the future.

Reply:
right, will do thanks

Reply:
done